shania_cox/alpha_challenges/the_basics.md

```diff
@@ -0,0 +1,163 @@
+
+## Question 4
+
+```ruby
+x = 2
+
+if x == 3
+  puts "sushi is tasty"
+elsif x > 0
+  puts "sushi is delicious"
+end
+```
+
+Imagine that you take the code from this question, save it to a file called `sushi.rb`, and run `ruby sushi.rb` in your Terminal.
+
+What would be the output? Explain your answer.
+
+**Answer 4:**
+The output would be "sushi is delicious" because since x = 2, the first part of the if statement would be skipped and then it would go to the second part of the if statement.
+
+## Question 5
+
+```ruby
+food = "walnut"
+
+if food == "walnut"
+  puts "I'm allergic!"
+end
+```
+
+If you ran this code, what would be the output? Explain your answer.
+
+**Answer 5:**
+Since food does have the value of "walnut" it would put out the statement "I'm allergic!"
+
+## Question 6
+
+```ruby
+  a = "4" + 10
+```
+
+What would happen here and why? How should I change this code to get an output of 14?
+
+**Answer 6:**
+If you were to run that, there would be an error because you can not add a string and an integer in Ruby. In order to change this, you could remove the quotation marks from the 4 or you could use the to_i method on the "4" to change it to an integer.
+
+
+## Question 7
+
+```ruby
+ turtles = ["Leonardo", "Raphael", "Michelangelo", "Donatello"]
+```
+
+What is the index value of "Michelangelo"?
+
+
+**Answer 7:**
+The index value of "Michelangelo" is 2.
+
+## Questions 8
+
+```ruby
+ jurassic_park = { "T-Rex" => 1, "Triceratops" => 4, "Velociraptors" => 6, "Humans" => ["Dr. Malcolm", "Dr. Grant"] }
+```
+
+* In Ruby, write code to access access the number of "Triceratops" in `jurassic_park`.
+
+**Answer 8A:**
+```ruby
+jurassic_park["Triceratops"]
+```
+
+* In Ruby, write code to add "Dr. Satler" to the "Humans" of `jurassic_park`.
+
+**Answer 8B:**
+```ruby
+jurassic_park["Humans"].push("Dr.Satler")
+```
+jurassic_park["Humans"].push("Dr. Salter") or jurassic_park["Humans"] << "Dr. Ziegler"
+
+## Question 9
+
+```ruby
+x = 0
+
+while x < 4
+  turtles = ["Leonardo", "Raphael", "Michelangelo", "Donatello"]
+  puts turtles[x]
+
+  x = x + 1
+end
+```
+
+* Examine the above code and imagine that you run it.
+* What do you expect the outcome will be? Explain your answer.
+
+**Answer 9:**
+I think that the code will output the strings in the array turtles. Since x = 0, it will print out the first string until it reaches the end of turtles. Once it reaches turtles, it will stop. Since the array length is 4
+
+## Question 11
+
+* Write a ruby method called `names` that returns an array with two names as strings inside of it.
+
+**Answer 11:**
+```ruby
+def names
+  puts ['Rofeeah', 'Shania']
+end
+```
+
+## Question 12
+
+* Write a ruby method called `caps` that takes a string as an argument and returns that string with all capital letters. (Hint: use the format from question 11 to write your method.)
+
+**Answer 12:**
+```ruby
+def caps("string")
+	puts string.upcase!
```

Comment:
Watch out for tabbing gremlins! 😄 Check those Sublime settings and make sure that you indent using spaces with a tab width of 2.